keyboards/handwired/dactyl_rah/keymaps/left/config.h

```diff
@@ -0,0 +1,19 @@
+/*
+Copyright 2020 Ralph Azucena <ralphie02@live.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+#define MASTER_LEFT
```

Comment:
This isn't needed as this is the default behavior.

Reply:
removed line and renamed the directory to "../keymaps/default" instead of "../keymaps/left".